shaka-lab-node/macos/log-wrapper.js

```diff
@@ -43,7 +43,11 @@ function main() {
   }
 
   // Extract arguments.
-  const [stdoutLogPath, stderrLogPath, pidPath, command, ...commandArgs] = args;
+  const stdoutLogPath = args[0];
```

Comment:
FMI, what's the difference? I believe `commandArgs` would be an empty in both cases array if args.length <= 4

Reply:
The only difference is that the new version fits in 80 columns.  It's just a minor style change.